models/eprop_learning_signal_connection_bsshslm_2020.h

```diff
@@ -77,9 +77,16 @@ weight    pA    :math:`B_{jk}`       1.0 Synaptic weight
 Recordables
 +++++++++++
 
-The following variables can be recorded:
+The following variables can be recorded. Note that since this connection lacks
```

Comment:
This file is almost precisely the same as the one without the "bsshslm_2020" in the name. Can the same file be used for both cases?